cala-ledger/tests/velocity.rs

```diff
@@ -191,3 +228,147 @@ async fn create_control_on_account_set() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn limit_children_accounts_via_grandparent_account() -> anyhow::Result<()> {
+    let pool = helpers::init_pool().await?;
+    let cala_config = CalaLedgerConfig::builder()
+        .pool(pool)
+        .exec_migrations(false)
+        .build()?;
+    let cala = CalaLedger::init(cala_config).await?;
+
+    let new_journal = helpers::test_journal();
+    let journal = cala.journals().create(new_journal).await.unwrap();
+
+    let tx_code = Alphanumeric.sample_string(&mut rand::rng(), 32);
+    let new_template = helpers::velocity_template(&tx_code);
+    cala.tx_templates().create(new_template).await.unwrap();
+
+    let velocity = cala.velocities();
+
+    let debit_limit = account_closing_limit(velocity, "DEBIT").await?;
+    let credit_limit = account_closing_limit(velocity, "CREDIT").await?;
+
+    let control = NewVelocityControl::builder()
+        .id(VelocityControlId::new())
+        .name("Account Closing")
+        .description("test")
+        .condition("context.vars.transaction.effective <= date('2024-12-31')")
```

Comment:
wouldn't it make more sense to have something like - effective is _before the current month_ or something else dynamic like that. Wouldn't want to have to create a new velocity every month.

Reply:
Yes, I think the expectation here would be that closing the books would be an explicit action.  I've sent a message to Guillermo to ask if some automatic close after the grace period would be ok to start with.